index.html

```diff
@@ -3465,6 +3488,13 @@ <h3><dfn>Switch To Frame</dfn></h3>
      <li><p>If <var>id</var> is less than 0 or greater than 2<sup>16</sup> – 1,
       return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
+     <li><p>If the <a>current browsing context</a> is <a>no
+           longer open</a>, return <a>error</a> with <a>error code</a> <a>no
+           such window</a>.
```

Comment:
This should be a `no such frame` error, right? Or should it only be the used when the current browsing context is not the top-level one?

Reply:
I think `no such frame` is only for the case where the specified frame can't be found, and `no such window` is always used for the case where the current browsing context can't be found.

Reply:
Ok, that makes sense. Otherwise also all the other commands would have to be changed too. And for those it doesn't make sense.